### PR TITLE
zellij: Add at 0.39.2

### PR DESCRIPTION
**Pull request by tarkah**

### CODE REVIEW

File: z/zellij/monitoring.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+releases:
+  id:
```
Comment (der-eismann) [inline z/zellij/monitoring.yml:2]: Drop a [300087](https://release-monitoring.org/project/300087/) here :slightly_smiling_face:

Comment (tarkah) [reply z/zellij/monitoring.yml]: Done, what does this relate to?

**Review by ermo — CHANGES_REQUESTED** [review]
Please use "zellij: Add at <version>" in the header. Makes it easier to follow monorepo activity in git logs.

We also traditionally put the summary key just above the description key for easy reference.

**Review by ermo — APPROVED** [review]
Optional: 

Play with adding e.g. `--release --config debug="full"` 

(per https://doc.rust-lang.org/cargo/reference/profiles.html#split-debuginfo and `man cargo`+search for `--config`)

I know you need zellij to feel productive so it's pointless to hold this up for any longer.

**Pull request merged**